docs/stream-aggregation/configuration/README.md

```diff
@@ -0,0 +1,116 @@
+  If match isn't set, then all the incoming samples are aggregated.
+  match also can contain a list of series selectors. Then the incoming samples are aggregated
+  if they match at least a single series selector.
+* [`interval`](?selfref=true#interval)  `(string: "", required)` - interval for the aggregation. The aggregated stats are sent to 
+  remote storage once per interval.
+* [`dedup_interval`](?selfref=true#dedup_interval) `(string:"")` - interval for de-duplication of input samples before the aggregation.
+  Samples are de-duplicated on a per-series basis. See [timeseries](https://docs.victoriametrics.com/keyconcepts#time-series) and [deduplication](#deduplication)
+  The deduplication is performed after [`input_relabel_configs`](#input-relabel-configs) relabeling is applied. By default, the deduplication is disabled
+  unless `-remoteWrite.streamAggr.dedupInterval` or `-streamAggr.dedupInterval` command-line flags are set.
+* [`staleness_interval`](?selfref=true#staleness_interval) `(string:2*interval)` - interval for resetting the per-series state if no new samples
+  are received during this interval for the following outputs:
+  * [`rate_avg`](./outputs/#rate_avg)
+  * [`rate_sum`](./outputs/#rate_sum)
+  * [`total`](./outputs/#total)
+  * [`total_prometheus`](./outputs/#total_prometheus)
+  * [`increase`](./outputs/#increase)
+  * [`increase_prometheus`](./outputs/#increase_prometheus)
+  * [`histogram_bucket`](./outputs/#histogram_bucket)
+  Check [staleness](#staleness) for more details.
+* [`no_align_flush_to_interval`](?selfref=true#no_align_flush_to_interval) `(bool: false)` - disables aligning of flush times for the aggregated data to multiples of interval.
+  By default, flush times for the aggregated data is aligned to multiples of interval.
+  For example:
+  * if [`interval:`](#interval) `1m` is set, then flushes happen at the end of every minute,
+  * if [`interval:`](#interval) `1h` is set, then flushes happen at the end of every hour
+* [`flush_on_shutdown`](?selfref=true#flush_on_shutdown) `(bool: false)` - instructs to flush aggregated data to the storage on the first and the last intervals
+  during vmagent starts, restarts or configuration reloads.
+  Incomplete aggregated data isn't flushed to the storage by default, since it is usually confusing.
+* [`without`](?selfref=true#without) `(list<string>: [])` - list of labels, which must be removed from the output aggregation.
+  See [aggregation by labels](#aggregating-by-labels)
+* [`by`](?selfref=true#by) `(list<string>: [])` - list of labels, which must be preserved in the output aggregation.
+  See [aggregation by labels](#aggregating-by-labels)
+* [`outputs`](?selfref=true#outputs) `(list<string>:[], required)` - list of aggregations to perform on the input data.
+  See [aggregation outputs](#outputs).
+* [`keep_metric_names`](?selfref=true#keep_input_names) `(bool: false)` - instructs keeping the original metric names for the aggregated samples.
```

Comment:
Is the link correct? the flag is `keep_metric_names` but refers to `#keep_input_names`